Mathlib/Algebra/Order/Ring/StandardPart.lean

```diff
@@ -81,22 +81,50 @@ instance : IsStrictOrderedRing (FiniteElement K) := by
 /-- The constructor for `FiniteElement`. -/
 protected def mk (x : K) (h : 0 ≤ mk x) : FiniteElement K := ⟨x, h⟩
 
-@[simp] theorem mk_zero (h : 0 ≤ mk (0 : K)) : FiniteElement.mk 0 h = 0 := rfl
-@[simp] theorem mk_one (h : 0 ≤ mk (1 : K)) : FiniteElement.mk 1 h = 1 := rfl
-@[simp] theorem mk_natCast {n : ℕ} (h : 0 ≤ mk (n : K)) : FiniteElement.mk (n : K) h = n := rfl
-@[simp] theorem mk_intCast {n : ℤ} (h : 0 ≤ mk (n : K)) : FiniteElement.mk (n : K) h = n := rfl
+@[simp] theorem mk_zero : FiniteElement.mk (0 : K) (by simp) = 0 := rfl
+@[simp] theorem mk_one : FiniteElement.mk (1 : K) (by simp) = 1 := rfl
+@[simp] theorem mk_natCast (n : ℕ) : FiniteElement.mk (n : K) (mk_natCast_nonneg n) = n := rfl
+@[simp] theorem mk_intCast (n : ℤ) : FiniteElement.mk (n : K) (mk_intCast_nonneg n) = n := rfl
 
 @[simp]
 theorem mk_neg {x : K} (h : 0 ≤ mk x) :
     -FiniteElement.mk x h = FiniteElement.mk (-x) (by rwa [mk_neg]) :=
   rfl
 
+@[simp]
+theorem mk_add_mk {x y : K} (hx hy) :
```

Comment:
I would make `x` and `y` explicit since `hx` and `hy` can be inferred from the LHS
```suggestion
theorem mk_add_mk (x y : K) (hx hy) :
```